packages/extension/package.json

```diff
@@ -21,7 +21,6 @@
     "@tanstack/react-query": "^4.36.1",
     "@tanstack/react-query-devtools": "^4.35.3",
     "classnames": "^2.5.1",
-    "content-scripts-register-polyfill": "^3.2.1",
```

Comment:
If I remember correctly, we moved the dependency to the webapp and dynamically imported it only when necessary.

Reply:
I think it was for firefox because it lacked some manifest v3 APIs, but yeah don't see any usages.